packs/journals/ancestries.json

```diff
@@ -433,7 +433,7 @@
wn to kashrishi as \"The Crystal Butterfly,\" and when a kashrishi child goes missing, their parents might pray the Crystal Butterfly uses the light of her wings to guide the child home. Besmara is also unusually likely to be worshipped by kashrishi, though her dogma among them is essentially a splinter faith that focuses on her primacy over sea monsters and storms with little regard to the piratical teachings for which most other ancestries know the goddess.</p><h2>Names</h2><p>The empathic sense of a kashrishi has its own \"mental fingerprint\" that's unique to each member of the species, and this mental fingerprint can be conveyed through psychic communications as a name. Kashrishi who adventure with non-kashrishi often adopt descriptive names that their companions can more easily speak and remember.</p><h3>Sample Names</h3><p>Climber, Firehorn, Guardian, Healer, Lantern, Mother, Scout, Softhand, Tempest, Warrior</p><h2>Kashrishi Mechanics</h2><p><strong>Rare</strong> Rare</p><p><strong>Hit Points</strong> 8</p><p><strong>Size</strong> Small</p><p><strong>Speed</strong> 25 feet</p><p><strong>Attribute Boosts</strong> Constitution, Free</p><p><strong>Languages</strong> Common, Kashrishi</p><p><strong>Additional Languages</strong> equal to your Intelligence modifier (if positive). Choose from Aquan, Celestial, Draconic, Sylvan, Terran, and any other languages to which you have access (such as the languages prevalent in your region)</p><p><strong>Traits</strong> Humanoid, Kashrishi</p><p><strong>Emphathic Sense</strong> Kashrishi are able to get a vague sense of the current emotional state of others within 15 feet of them. This manifests as a general feeling of happiness, anger, or fear, without any specific details. For those with whom kashrishi have an emotional connection, physical touch can heighten this sense to greater degrees of detail and understanding, increasing with the strength of their bond. This also functions as an imprecise sense alerting you to the presence, but not the precise location, of non-mindless creatures within the area. You gain a +2 circumstance bonus on Perception checks to Sense Motive against non-mindless creatures within the area of your empathic sense</p><p><strong>Emphathic Sense</strong> Your horn reacts to psychic energy by softly glowing. The horn emanates dim light in a 10-foot emanation until the start of your next turn whenever you use an occult action you gained from an ancestry feat, cast an innate occult spell, or Cast a Spell that has the mental trait.</p><h2>Kashrishi Heritages</h2><p>A kashrishi's heritage reflects the unique evolutionary qualities they've adapted for their chosen environments. Choose one kashrishi heritage at 1st level.</p><p>@UUID[Compendium.pf2e.heritages.Item.Athamasi]</p><p>@UUID[Compendium.pf2e.heritages.Item.Lethoci]</p><p>@UUID[Compendium.pf2e.heritages.Item.Nascent]</p><p>@UUID[Compendium.pf2e.heritages.Item.Trogloshi]</p><p>@UUID[Compendium.pf2e.heritages.Item.Xyloshi]</p>",
```

Comment:
```suggestion
                "content": "<p>Kashrishi make their homes in remote areas of the world. These quiet beings have stout, durable frames and distinctive crystalline horns. Their inherent psychic abilities make them natural empaths but also occasionally burden them with the unceasing thoughts of their neighbors.</p><p>Kashrishi occupy a unique evolutionary branch native to the lands bordering the northern stretches of the Obari Ocean. Resembling halfling-sized bipedal rhinoceroses with the occasional odd insectile feature, kashrishi adapt to their environs with unusual efficiency, using a combination of rapid physical evolutions and inherent psychic powers. Kashrishi are often mistakenly referred to as having hive minds, though they're actually natural empaths, capable of discerning a creature's emotional state and impulses through proximity. This near-oracular behavior occasionally leads to misunderstandings with humanoids who think a kashrishi is reading their thoughts or otherwise magically influencing the conversation, though peoples more familiar with kashrishi come to value their unique insights and intuitive diplomacy.</p><p>Kashrishi have an atypical level of control over their physical forms, thanks to their psychic powers, and can evolve new features over the course of a single generation. Typically, these features are intentionally cultivated to help deal with a particular environmental obstacle, such as a change in weather patterns, or if an invasive species affects available food supplies near their settlements.</p><p>Kashrishi often tailor their evolutions to match the most successful creatures in their environment. Whether a testament to the resilience of insects or simply a quirk of their environment, many kashrishi evolutions are directly inspired by creatures like the rhinoceros beetle or giant water bug.</p><p>If you want a character that's visually distinct, able to naturally access the occult powers of the mind, and great at quickly making friends wherever they go, you should play a kashrishi.</p><h2>You Might...</h2><ul><li>Be easily taken aback by people who are particularly loud, expressive, or emotional.</li><li>Prefer the peaceful quiet of remote islands, treetops, and caverns.</li><li>Act as a parental figure for the more excitable among your companions.</li></ul><h2>Others Probably...</h2><ul><li>Underestimate your strength and resilience.</li><li>Mistake you for being unsociable when you're actually taking time to process mental and physical cues of which they're completely unaware.</li><li>Value you for your patience and insights.</li></ul><h2>Physical Description</h2><p>Kashrishi have short but wide frames with remarkable strength for their size. All have signature crystalline horns, though these can grow in a variety of configurations. The most common kashrishi, xyloshis, have a large primary horn and between one and three smaller horns, typically arranged linearly from the tip of their nose and spanning along their nose and brow.</p><p>While most kashrishi have light sandy or gray skin tones, their coloration can also echo the tones of their crystalline horns; sapphire, ruby, or emerald patterns aren't so uncommon as to cause another kashrishi to take notice. Kashrishi with strong connections to specific creatures with whom they share their home environments might take on skin patterns or colorations that mirror those creatures. Some stories even talk about kashrishi with wings and butterfly patterns that cover their bodies, though no such kashrishi has been seen in the Impossible Lands for at least a century.</p><h2>Society</h2><p>Despite their small stature, kashrishi tend to be terrible at hiding. Their crystalline horns are likely to flare with magical light when they exercise their innate magical powers, and they're raised in an environment where their parents or guardians can almost always find them by feeling for their emotions. As a result, kashrishi tend to be excessively honest and kind, avoiding harmful deceptions but also employing enough tact to ensure they don't evoke unpleasant emotional reactions from others.</p><p>Kashrishi lack a clear concept of charity, not because they're cruel or inconsiderate, but rather because their worldview is inherently inclusive of the people around them. Rare indeed is the kashrishi who exercises casual cruelty or who leaves another member of the community in need. Such actions lead to emotional reactions that cause turmoil for all kashrishi in the vicinity.</p><p>Kashrishi enjoy various team games, employing games of trivia as measures of both knowledge and psychic ability. They also appreciate games that involve bluffs and double-bluffs with blind cards, testing their abilities to keep their minds clear and their thoughts organized. Kashrishi don't generally condone the act of gambling with outsiders, as many find it challenging to deal with the implications of their psychic abilities, even if they strive to keep their powers secret.</p><p>Kashrishi generally engage in monogamous relationships with no real preference toward any particular gender pairing. The height of intimacy for a kashrishi is opening their mind fully to the psychic link of another kashrishi, an act much more personal than anything physical. This act doesn't let kashrishi know whether someone is perfectly compatible with them or that the two mesh entirely—simply that from moment to moment, their inner feelings are more accessible and open, so communication becomes not a guessing game but an entirely wholesome, true experience. Some kashrishi believe that psychic communication should be held exclusively for these intimate moments.</p><p>Kashrishi can be found in small groups on the island of Jalmeray and in larger communities occupying the many smaller islands surrounding it. Kashrishi in the Inner Sea region rarely venture beyond the Impossible Lands, though some of their oldest stories speak of a harrowing voyage across the Obari Ocean when their kind fled some long-past disaster.</p><p>Kashrishi have no natural enemies. They host pirates and merchants with equal hospitality, so long as that hospitality is respected by their guests. While kashrishi are exceptionally tolerant of the quirks and foibles of other species, they draw a hard line at anyone bringing outside conflicts into their communities. More than one canny pirate captain docks their ship at a kashrishi island after a particularly heated conflict, knowing the pirate hunters and military vessels seeking them won't encroach on kashrishi lands. Escaped criminals hoping to find a new lease on life might also hide in a kashrishi community, doing whatever they can to assist their hosts until whoever is hunting them gives up the chase.</p><h2>Alignment and religion</h2><p>Most kashrishi are good, and almost instinctively kind because of their empathic abilities and tight-knit communities. Kashrishi are rarely interested in religion, though a few deities are more likely to be worshipped by kashrishi than others. Desna is known to kashrishi as \"The Crystal Butterfly,\" and when a kashrishi child goes missing, their parents might pray the Crystal Butterfly uses the light of her wings to guide the child home. Besmara is also unusually likely to be worshipped by kashrishi, though her dogma among them is essentially a splinter faith that focuses on her primacy over sea monsters and storms with little regard to the piratical teachings for which most other ancestries know the goddess.</p><h2>Names</h2><p>The empathic sense of a kashrishi has its own \"mental fingerprint\" that's unique to each member of the species, and this mental fingerprint can be conveyed through psychic communications as a name. Kashrishi who adventure with non-kashrishi often adopt descriptive names that their companions can more easily speak and remember.</p><h3>Sample Names</h3><p>Climber, Firehorn, Guardian, Healer, Lantern, Mother, Scout, Softhand, Tempest, Warrior</p><h2>Kashrishi Mechanics</h2><p><strong>Rarity</strong> Rare</p><p><strong>Hit Points</strong> 8</p><p><strong>Size</strong> Small</p><p><strong>Speed</strong> 25 feet</p><p><strong>Attribute Boosts</strong> Constitution, Free</p><p><strong>Languages</strong> Common, Kashrishi</p><p><strong>Additional Languages</strong> equal to your Intelligence modifier (if positive). Choose from Aquan, Celestial, Draconic, Sylvan, Terran, and any other languages to which you have access (such as the languages prevalent in your region)</p><p><strong>Traits</strong> Humanoid, Kashrishi</p><p><strong>Emphathic Sense</strong> Kashrishi are able to get a vague sense of the current emotional state of others within 15 feet of them. This manifests as a general feeling of happiness, anger, or fear, without any specific details. For those with whom kashrishi have an emotional connection, physical touch can heighten this sense to greater degrees of detail and understanding, increasing with the strength of their bond. This also functions as an imprecise sense alerting you to the presence, but not the precise location, of non-mindless creatures within the area. You gain a +2 circumstance bonus on Perception checks to Sense Motive against non-mindless creatures within the area of your empathic sense</p><p><strong>Emphathic Sense</strong> Your horn reacts to psychic energy by softly glowing. The horn emanates dim light in a 10-foot emanation until the start of your next turn whenever you use an occult action you gained from an ancestry feat, cast an innate occult spell, or Cast a Spell that has the mental trait.</p><h2>Kashrishi Heritages</h2><p>A kashrishi's heritage reflects the unique evolutionary qualities they've adapted for their chosen environments. Choose one kashrishi heritage at 1st level.</p><p>@UUID[Compendium.pf2e.heritages.Item.Athamasi]</p><p>@UUID[Compendium.pf2e.heritages.Item.Lethoci]</p><p>@UUID[Compendium.pf2e.heritages.Item.Nascent]</p><p>@UUID[Compendium.pf2e.heritages.Item.Trogloshi]</p><p>@UUID[Compendium.pf2e.heritages.Item.Xyloshi]</p>",
```